test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -300,6 +300,7 @@ function ExecuteRestMethod {
     $verbosePreferenceSave = $VerbosePreference
     $VerbosePreference = 'Continue'
     try {
+        $ENV:__PWSH_TESTOUTPUT = 'true'
```

Comment:
Why do we need this?

Reply:
Noted above. If it really bothers you what I can do is include the encoding information into the debug output and adjust this test to look for it that way.